envs/monkey_zoo/docs/zoo_network.md

```diff
@@ -1156,7 +1157,43 @@ setting:
 <table>
 <thead>
 <tr class="header">
-<th><p><span id="_Toc536021491" class="anchor"></span>Nr. <strong>250 MonkeyIsland</strong></p>
+<th><p><span id="_Toc536021491" class="anchor"></span>Nr. <strong>3-70 Chrome Credentials Collector</strong></p>
+<p>(10.2.3.70)</p></th>
+<th>(Vulnerable)</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>OS:</td>
+<td><strong>Ubuntu 20.04LTS</strong></td>
+</tr>
+<tr class="even">
+<td>Software:</td>
+<td>Chromium (snap)</td>
+</tr>
+<tr class="odd">
+<td>SSH</td>
+<td>Port: 22</td>
+</tr>
+<tr class="even">
+<td>Notes:</td>
```

Comment:
Add a note that only the files are moved and no chrome is or should be installed.